pkg/scheduler/framework/plugins/nodelabel/node_label.go

```diff
@@ -110,7 +110,7 @@ func (pl *NodeLabel) Filter(ctx context.Context, _ *framework.CycleState, pod *v
 func (pl *NodeLabel) Score(ctx context.Context, state *framework.CycleState, pod *v1.Pod, nodeName string) (int64, *framework.Status) {
 	nodeInfo, err := pl.handle.SnapshotSharedLister().NodeInfos().Get(nodeName)
 	if err != nil || nodeInfo.Node() == nil {
-		return 0, framework.NewStatus(framework.Error, fmt.Sprintf("getting node %q from Snapshot: %v, node is nil: %v", nodeName, err, nodeInfo.Node() == nil))
+		return 0, framework.AsStatus(fmt.Errorf("getting node %q from Snapshot: %w, node is nil: %v", nodeName, err, nodeInfo.Node() == nil))
```

Comment:
We don't need the check `nodeInfo.Node() == nil` anymore. Also, `%w` should always be at the end of the string.